fairscale/optim/oss.py

```diff
@@ -328,6 +328,9 @@ def consolidate_state_dict(self, recipient_rank: int = 0) -> None:
         should_collect_state = self.rank == recipient_rank or recipient_rank == -1
         should_send_state = (self.rank != recipient_rank and recipient_rank != -1) or recipient_rank == -1
```

Comment:
This could be simplified to:
```suggestion
        should_send_state = self.rank != recipient_rank
```

Or:

```suggestion
        should_send_state = recipient_rank == -1 or self.rank != recipient_rank
```

Reply:
Agreed, thanks Mandeep !